### PR TITLE
Add possibility to edit files on a message

### PR DESCRIPTION
**Pull request by Gugu7264**
## Summary

Adds multiple arguments to the `Message.edit()` function, now allowing to edit files when editing a message.

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by vcokltfre — COMMENTED** [review]
This all looks good, but I cannot test it right now. Take this as a semi-approval, but should still be approved by people who have properly tested it.

**Review by TAG-Epic — APPROVED** [review]
Code looks good to me, not tested.

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
Versionadded should be added

**Comment by Gugu7264** [comment]
> Versionadded should be added

What version should it be?

File: nextcord/message.py [hunk 1]
```diff
@@ -1237,13 +1237,19 @@ async def edit(
             the view is removed.
         file: Optional[:class:`File`]
             A new file to add to the message.
+
+            .. versionadded:: 2.0
         files: Optional[List[:class:`File`]]
             A list of new files to add to the message.
+
+            .. versionadded:: 2.0
         append_files: Optional[:class:`bool`]
```
Comment (koala9712) [inline nextcord/message.py:1246]: Include default value?
lgtm, tested!

Comment (Gugu7264) [reply nextcord/message.py:1246]: There is no default value though, updated the docs a little bit though, does that seem correct?

Comment (koala9712) [reply nextcord/message.py:1246]: Just wondering what happens if I don't pass in `append_files`
Defaults to True, hence including that would be a good idea.

Comment (Gugu7264) [reply nextcord/message.py:1246]: Default for `append_files` is already there??

Comment (Gugu7264) [reply nextcord/message.py:1246]: "If set to True (default)"

**Review by TAG-Epic — APPROVED** [review]
Not tested

**Review by TAG-Epic — APPROVED** [review]
Tested

**Pull request merged**